### PR TITLE
Preprocess output as markdown

### PR DESCRIPTION
**Pull request by mkandler**
Also added some extra line breaks to make it easier to parse on the frontend

### CODE REVIEW

**Comment by mkandler** [comment]
Current dependencies on/for this PR:
* **#5** <a href="https://app.graphite.dev/github/pr/run-house/notebooks/5?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/run-house/notebooks/5?utm_source=stack-comment).

File: nbconvert_config.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+c = get_config()
+
+# c.NbConvertApp.notebooks = ['./examples/inference-stable-diffusion-flan-t5.ipynb']
```
Comment (carolineechen) [inline nbconvert_config.py:3]: we can remove these comments right

**Review by carolineechen — APPROVED** [review]
🎉

File: nbconvert_config.py (hunk 1 above)
Comment (mkandler) [reply nbconvert_config.py]: Good catch, forgot to clean that up

**Pull request merged**